components/brave_rewards/resources/ui/components/walletWrapper/index.tsx

```diff
@@ -673,7 +678,11 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
               ? <LoginMessage>
                   <LoginMessageText>
                     <b>{getLocale('loginMessageTitle')}</b>
-                    <p dangerouslySetInnerHTML={{ __html: getLocale('loginMessageText') }} />
+                    <p>
```

Comment:
also not sure where to test this; do i have to link an uphold account first?

Reply:
nvm, i think i figured it out. make sure you haven't claimed a grant, go to brave://rewards/, and click verify wallet. You should see a message that says you need minimum 25 BAT, try again later.